snuba/web/db_query.py

```diff
@@ -191,6 +194,8 @@ def execute_query(
         clickhouse_query_settings["load_balancing"] = "in_order"
         clickhouse_query_settings["max_threads"] = 1
 
+    stats["parallel_read"] = "max_parallel_replicas" in clickhouse_query_settings
```

Comment:
Why did you call this `parallel_read`? Why not leave it `max_parallel_replicas` for clarity? Also, why not set the actual value?

Reply:
done